### PR TITLE
Add 1-29 release branch support

### PR DESCRIPTION
**Pull request by abhay-krishna**
This PR adds EKS-D 1-29 release branch support to the build-tooling projects.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: projects/kubernetes-sigs/image-builder/Makefile [hunk 1]
```diff
@@ -67,6 +67,7 @@ ADDITIONAL_PAUSE_1-25_FROM=1-24
 ADDITIONAL_PAUSE_1-26_FROM=1-26
 ADDITIONAL_PAUSE_1-27_FROM=1-27
 ADDITIONAL_PAUSE_1-28_FROM=1-28
+ADDITIONAL_PAUSE_1-29_FROM=1-28
```
Comment (jaxesn) [inline projects/kubernetes-sigs/image-builder/Makefile:70]: did you verify this based on the above rules?

Comment (abhay-krishna) [reply projects/kubernetes-sigs/image-builder/Makefile:70]: Kubernetes 1.29 had [pause version 3.9](https://github.com/kubernetes/kubernetes/blob/v1.29.0-rc.1/cmd/kubeadm/app/constants/constants.go#L423) in the constants.go, and both [1.28](https://github.com/kubernetes/kubernetes/blob/v1.28.3/build/pause/Makefile#L20) and [1.29](https://github.com/kubernetes/kubernetes/blob/v1.29.0-rc.1/build/pause/Makefile#L20) had 3.9 as the pause version in the Makefile. So I guess 1-28 should also work? But I can change it to 1-29.

**Review by jaxesn — APPROVED** [review]

**Pull request merged**